app/models/concerns/webauthn_concern.rb

```diff
@@ -0,0 +1,27 @@
+module WebauthnConcern
+  extend ActiveSupport::Concern
+
+  included do
+    has_many :webauthn_credentials, dependent: :destroy
+
+    after_initialize do
+      self.webauthn_id ||= WebAuthn.generate_user_id
+    end
+  end
+
+  def webauthn_options_for_create
+    WebAuthn::Credential.options_for_create(
+      user: {
+        id: webauthn_id,
+        name: display_id
```

Comment:
This name field is [required](https://github.com/cedarcode/webauthn-ruby/blob/9544853522a351371d466c966c06ee7b24d18d9a/lib/webauthn/public_key_credential/entity.rb#L10) by the webauthn client but we don't actually use it anywhere. 

I'm not sure if we should leave this `name` as the display_id or switch to the user id given that the display_id can change. According to the [docs](https://www.w3.org/TR/webauthn-2/#dom-publickeycredentialentity-name) this should be a `human palatable identifier for a user` so display_id seems more right here. But if we ever did want to use display this field in the future (idk why we would) it could be incorrect if the user had changed their handle since they created the webauthn key.